docs/source/installation.rst

```diff
@@ -0,0 +1,110 @@
+.. _installation:
+
+Installation instructions
+=========================
+
+Note: T3 was only tested on Linux (Ubuntu_ 18.04.1 LTS) and Mac machines. We don't expect it to work on Windows.
+T3 can be installed on a server, as well as on your local desktop / laptop, submitting jobs to the server/s.
```

Comment:
T3 doesn't require RMG to be installed, it's a conda dependency

Reply:
My bad. I didn’t pay attention to the environment file